change/@microsoft-teams-js-2339868b-2814-483b-85e4-2d2075bd022e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Reverted media.ts to v1 state and made necessary changes in unit tests and test-app",
```

Comment:
Can you re-word this to: "Updated `media` namespace to work as it did in v1 along with necessary changes to unit tests and teams-test-app" please?